docs/modules/customize/pages/oauth.adoc

```diff
@@ -1,10 +1,10 @@
-# OAuth
+= OAuth
 
-Decidim is both and OAuth client (via [omniauth](https://github.com/omniauth/omniauth)) and an OAuth provider (via [doorkeeper](https://github.com/doorkeeper-gem/doorkeeper)).
+Decidim is both and OAuth client (via https://github.com/omniauth/omniauth[omniauth]) and an OAuth provider (via https://github.com/doorkeeper-gem/doorkeeper[doorkeeper]).
 
-Check the [Social Providers](https://github.com/decidim/decidim/blob/master/docs/services/social_providers.md) document to check the client configuration.
+Check the https://github.com/decidim/decidim/blob/master/docs/services/social_providers.md[Social Providers] document to check the client configuration.
```

Comment:
If PR #6607  is merged, then this section would have to be updated too..

Reply:
Ok, I've already migrated that (I think :sweat_smile:)